### ISSUE TITLE
react-native-debugger starts fail on Fedora 37

### ISSUE THREAD

**Issue by rafaeldellaquila** [jhen0409/react-native-debugger#749]
React Native Debugger app version: **react-native-debugger-0.13.0-1.amd64.rpm**
React Native version: **"react-native": "0.70.5"**
Platform: **Android**
Is real device of platform: **Yes**
Operating System: **Fedora 37**

I installed the react-native-debug normally but when i try to start the program i receive this crash from Electron:

```bash
Starting listen set-debugger-loc request on port 40949
Will save port to `$HOME/.rndebugger_port` file
(node:21145) ExtensionLoadWarning: Warnings loading extension at /usr/lib/react-native-debugger/resources/webextension/: Unrecognized manifest key 'author'. Unrecognized manifest key 'page_action'. Unrecognized manifest key 'short_name'. Permission 'tabs' is unknown or URL pattern is malformed. 
(node:21145) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
[21145:0202/114528.660661:FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.
[1]    21145 trace trap (core dumped)  react-native-debugger
```

I looked for a solution, but I couldn't solve this... Anyone can help me?

**Comment by 4ndrs** [comment]
> FATAL:gpu_data_manager_impl_private.cc(445)] GPU process isn't usable. Goodbye.

I had the same issue on my Gentoo box. To make it run I had to add the `--no-sandbox` flag to the debugger:
```console
$ ./react-native-debugger --no-sandbox
```

**Comment by jhen0409** [comment]
The issue for Linux should fixed by [v0.14.0](https://github.com/jhen0409/react-native-debugger/releases/tag/v0.14.0) (bumped Electron version). I've confirmed with Ubuntu 22.04.

If you still seeing the problem on Fedora, feel free to request a re-open.
